chrome/content/zotero/bibliography.js

```diff
@@ -195,10 +212,11 @@ var Zotero_File_Interface_Bibliography = new function() {
 		
 		// Change label to "Citation" or "Note" depending on style class
 		if(document.getElementById("citations")) {
+			var label = "";
```

Comment:
When you want to constrain it to the outer block, as is the case here, defining it before with ``let`` is best.

When ``var`` is necessary, I personally usually just repeat it, except maybe when there are a bunch of declarations or otherwise complex logic. For just a couple of them, avoiding the extra line seems more important than whatever case can be made against repeated declarations. Doesn't really matter, though.